level_five.gd

```diff
@@ -21,15 +21,21 @@ func _ready():
 	$Timer.start(1)
 
 
+func _process(_delta):
+	$CanvasLayer/hud/HBoxContainer2/AvailableTurrets.text = "Available Turrets: " + str($player.all_available_turrets())
```

Comment:
It doesn't really matter for this project, but it is worth noting this causes the text to be updated and recalculated 60+ times a second, as compared to adding it to the `on_timer_timeout` below that will cause it to only update 1 time per second.

The godot docs recommend moving anything that doesn't need to happen every frame into a method that is a callback to a signal. (ie. if we want the turrets count to always be up to date, and not a second behind, we could emit `turret-added` and `turret-removed` signals, and update the text then.

Reply:
Yeah I moved it out of the on_timer_timeout section because it didn't feel responsive when placing a turret

Though, I think rather than having it run every frame, we can trigger the text to update when the actual turret is placed. So yeah, I reject this PR until that change is made

Reply:
fixd